hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -25,9 +25,12 @@ function HackathonForm(props: IProps) {
   const [endDateValue, setEndDateValue] = useState<Date | null>(new Date())
   const [hackathonTitle, setHackathonTitle] = useState('')
   const [hackathonSlug, setHackathonSlug] = useState('')
-  const [DescriptionValue, onChange] = useState('')
   const [hackathon, setHackathon] = useState<Hackathon>({} as Hackathon)
 
+  const initialValue =
```

Comment:
a better name for this would be good something like `hackathonDescriptionHint`

Reply:
if you change it to a use a placeholder, then you could just in-line the value there and don't bother with a constant

Reply:
also with text editors in general allowing HTML can be... scary to say the least.  My personal view on the matter is to permit some form of markdown as it allows for nice pretty formatted things, but not so much freedom that you can build a scam website in a form and possibly trick people into doing something dumb

Reply:
The RTE is only accessible for admins so I think this would be ok 
Admins as we have them could also directly manipulate the source code so no need for hacky html tricks